perf_bench_test.go

```diff
@@ -24,127 +24,567 @@ package main
 
 import (
 	"context"
+	"encoding/csv"
 	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
 	"strconv"
+	"strings"
 	"sync"
+	"sync/atomic"
 	"testing"
 	"time"
 
+	"github.com/ease-lab/vhive/metrics"
+	"github.com/ease-lab/vhive/profile"
+	"github.com/montanaflynn/stats"
 	log "github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	isColdStart     = flag.Bool("coldStart", false, "Profile cold starts (default is false)")
-	vmNum           = flag.Int("vm", 2, "The number of VMs")
-	targetReqPerSec = flag.Int("requestPerSec", 2, "The target number of requests per second")
-	executionTime   = flag.Int("executionTime", 2, "The execution time of the benchmark in seconds")
+	// shared arguments
+	warmUpTime   = flag.Float64("warmUpTime", 5, "The warm up time before profiling in seconds")
+	coolDownTime = flag.Float64("coolDownTime", 1, "The cool down time after profiling in seconds")
```

Comment:
lines 50 and 51 should be swapped places

Reply:
Done